pkg/stream/input_sqs.go

```diff
@@ -21,13 +21,14 @@ type SqsInputSettings struct {
 	RedrivePolicy     sqs.RedrivePolicy     `cfg:"redrive_policy"`
 	Client            cloud.ClientSettings  `cfg:"client"`
 	Backoff           cloud.BackoffSettings `cfg:"backoff"`
+	Unmarshaller      string                `cfg:"unmarshaller" default:"msg"`
 }
 
 type sqsInput struct {
 	logger      mon.Logger
 	queue       sqs.Queue
 	settings    SqsInputSettings
-	unmarshaler MessageUnmarshaler
+	unmarshaler UnmarshallerFunc
```

Comment:
typo in `unmarshaler`?

Reply:
According to Java notation, that's the correct way to write it: https://docs.oracle.com/javase/7/docs/api/javax/xml/bind/Unmarshaller.html